packages/common/src/ee/serviceAccounts/scopes.ts

```diff
@@ -0,0 +1,27 @@
+export const ProjectScope = {
+    MANAGE: 'project:manage',
+} as const;
+
+export const ContentScope = {
+    MANAGE: 'content:manage',
+} as const;
+
+export const ScimScope = {
+    MANAGE: 'scim:manage',
+} as const;
+
+// Organization scopes for backwards compatibility
+export const OrgScope = {
```

Comment:
how do we move to getting rid of this?

Reply:
Was being cautious about a breaking change. I haven't dug through the data for usage yet, so I kept it for the time being. I suppose if it broke things for customers that adopted early, they can just create a new service account. I'll drop it.

Reply:
I'm fine with changing this to `role:admin` and dropping most of the PR. We'll have to think a bit more on mapping scopes properly and I'm lacking a lot of context for all the things.